Source/VSProps/Base.props

```diff
@@ -33,13 +33,14 @@
         Note: Directory containing source file being compiled is always searched first.
       -->
       <AdditionalIncludeDirectories>$(CoreDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <AdditionalIncludeDirectories>$(ExternalsDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
This line is as eval as doing so on linux. However without any system library, it likely won't matter.